frontend/src/ui/strings.json

```diff
@@ -318,7 +318,115 @@
+    "hiThisIs": "Ciao! Questo è",
+    "hostedBy": "Ospitato da",
+    "hostFee": "Commissioni dell'ospite",
+    "howDoYouWantToBeShown": "Come dovremmo visualizzarti sulla pagina? (Lascia vuoto per rimanere anonimo)",
+    "howWeSpend": "Guarda come lo spendiamo",
+    "IamAProudSupporter": "Sono un orgoglioso sostenitore di questi collettivi",
+    "ifDifferentThanAbove": "se diverso da quanto sopra",
+    "joinAndFulfil": "Unisciti a noi e aiutaci a completare la nostra missione",
+    "latestActivities": "Ultime donazioni",
+    "latestExpenses": "Le nostre ultime spese",
+    "member": "membro",
+    "members": "membri",
+    "membersWallText": "Membri e finanziatori supportano orgogliosamente questo collettivo e ci permettono di continuare a perseguire i nostri obiettivi. Non saremmo qui se non fosse per queste fantastiche persone. Grazie infinite.",
+    "membersWallTitle": "Questo è possibile grazie a te.",
+    "menuExpensesAndActivities": "Spese e Attività",
+    "menuMembersWall": "Muro dei Membri",
+    "menuWho": "Chi siamo",
+    "menuWhy": "Perché unirti?",
+    "missionTo": "Siamo in missione per",
+    "month": "mese",
+    "monthly": "Mensilmente",
+    "name": "Nome",
+    "note": "Note",
+    "nowOnBackersWall": "Ora sei sul Muro dei nostri sostenitori!",
+    "ocCommission": "Commissioni di OpenCollective",
+    "one-time": "Una volta",
+    "openCollective": "Open Collective",
+    "optional": "opzionale",
+    "other": "Altro",
+    "otherSimilarCollectives": "Altri Collettivi Similari",
+    "paid": "pagato",
+    "pay": "Paga",
+    "pending": "in attesa",
+    "payoutMethods-alreadyReimbursed": "Già rimborsato",
+    "payoutMethods-other": "Altro (per favore, fornisci delle istruzioni)",
+    "paypalEmail": "Email di PayPal",
+    "per": "per",
+    "plzConfirmYour": "Per favore, conferma il tuo",
+    "processingFee": "commissioni per l'elaborazione",
+    "proudMember": "Appartengo con orgoglio a questi collettivi",
+    "reimbursementMethod": "Metodo di rimborso",
+    "rejected": "respinto",
+    "resetDefaultDistribution": "Azzera distribuzione predefinita",
+    "save": "Salva",
+    "scrollDown": "Scorri in basso per scoprire di più.",
+    "since": "da",
+    "sponsor": "Finanziatore ufficiale",
+    "sponsors": "Finanziatori",
+    "submitExpense": "Presenta spesa",
+    "submittedBy": "Presentato da",
+    "subtotal": "Subtotale",
+    "thankyou": "Grazie per il tuo supporto",
+    "thatProvideYearlyBudget": "che ci procura un budget annuale di",
+    "total": "Totale",
+    "twitter": "Twitter",
+    "uploadReceipt": "Carica ricevuta (foto o PDF)",
+    "vat": "IVA",
+    "weAre": "Noi siamo",
+    "WeAreProudSupporters": "Siamo orgogliasi sostenitori di questi collettivi",
+    "website": "Sito web",
+    "weHave": "Noi abbiamo",
+    "year": "anno",
+    "yearly": "Annualmente",
+    "youDeciceDistributeYour": "Tu decidi come distribuire il tuo"
```

Comment:
Unrelated to IT language, but `youDeciceDistributeYour` seems to have a typo (decice)